cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_data_resources.go

```diff
@@ -506,7 +506,7 @@ func (s *Server) updatePkgInstall(ctx context.Context, cluster, namespace string
 // getAppUsedGVs returns the list of GVs used by the given app, falling back to pre 0.47 Kapp version behavior with regards to suffixes
 func getAppUsedGVs(appsClient ctlapp.Apps, packageId string, namespace string, useNewCtrlAppSuffix bool) ([]schema.GroupVersion, ctlapp.App, error) {
 	// We first try to fetch the app using the suffixed name (kapp >= 0.47)
-	appName := fmt.Sprintf("%s%s", packageId, ctlapp.AppSuffix)
```

Comment:
Why changing this? It should the one available in the kappctrl dependency at compile time, no?

Reply:
It turns out that this looks for a name ending in `.apps.k14s.io` (value in constant), but the actual identifier ends in `.app`.

Reply:
Ahh, ok, ok... what a pity we don't have any constant to rely on in the kappctrl pkg :(

Reply:
More info in [Rename KC owned apps from `x-ctrl` to `x.app` or `x.pkgr`](https://github.com/vmware-tanzu/carvel-kapp-controller/pull/665).